package.json

```diff
@@ -36,7 +36,7 @@
   "license": "MIT",
   "peerDependencies": {
     "react": "^16.0",
-    "react-native": ">=0.57 <=0.59"
+    "react-native": ">=0.57 <=0.61"
```

Comment:
[imo] I think we should just leave the upper limit empty, since this part will need to be changed everytime there is an update to react-native version.